components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -769,24 +769,44 @@ def start_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts
     logger.info(f"Started {component_name}.")
 
 
-def start_log_viewer_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts):
+def start_log_viewer_webui(
+    instance_id: str,
+    clp_config: CLPConfig,
+    container_clp_config: CLPConfig,
+    mounts: CLPDockerMounts,
+):
     component_name = LOG_VIEWER_WEBUI_COMPONENT_NAME
     logger.info(f"Starting {component_name}...")
 
     container_name = f"clp-{component_name}-{instance_id}"
     if container_exists(container_name):
         return
 
-    log_viewer_webui_logs_dir = clp_config.logs_directory / component_name
```

Comment:
We can add `log_viewer_webui_logs_dir` back once logging to file is enabled in log-viewer-webui-server.